auto_nag/scripts/component.py

```diff
@@ -62,6 +74,9 @@ def get_bugs(self, date='today', bug_ids=[]):
         # Retrieve history.
         self.retrieve_history(bugs)
 
+        # Remove bugs for which somebody has modified the product/component.
+        self.remove_using_history(bugs)
```

Comment:
We can probably avoid that in improving bz query (and so less bugs to retrieve):
not (f1='product', o1='changedafter', v1=start_date) and not(f2='component', o2='changedafter', v2=start_date))
and by the way, you can use (f=creation_ts, o=greaterthan, v=start_date) to get bugs created after start_date

Reply:
Done with 4be84d9c5db8d74226695ec68abf9a73ea2e92d8.